### ISSUE TITLE
Allow `select` `value` attribute in Svelte

### ISSUE THREAD

**Issue by kmoschcau** [markuplint/markuplint#1565]
- Markuplint version: `4.3.0`
- Parser lang: Svelte
- Node.js version: `20.11.0`
- OS: Archlinux

## Describe the bug

Normally the `value` attribute of a `select` shouldn't be used in markup. (Selected values are given via its options' `selected` attribute.) However Svelte does some magic here and allows binding to that `value` to set and read the selected `option` of a `select`. See [here](https://learn.svelte.dev/tutorial/select-bindings) for reference. I filed a similar issue here: https://github.com/markuplint/markuplint/issues/1262

## Code Example

```svelte
<script>
  let selected = { id: 1 };
</script>

<select bind:value={selected}><option value="1">Foo</option></select>
```

## Steps To Reproduce

1. Run markuplint over the above code.

## The current behavior

Markuplint returns the following error:

> error: The "value" attribute is disallowed (invalid-attr)

## The expected behavior

Markuplint should not return an error.
